### ISSUE TITLE
enigma: Config to auto-download keys to verify?

### ISSUE THREAD

**Issue by lrosenman** [roundcube/roundcubemail#5314]
Is there a way to automatically have enigma download keys from a keyserver?

**Comment by alecpl** [comment]
No, there's no such option. I don't think we should do this automatically. However, when we implement key server support we could add a [Find the key] button to simplify the process of finding the key and re-verifying the signature.

**Comment by lrosenman** [comment]
I like the idea of find the key, and allowing folks to set a keyserver or set of keyservers.

**Comment by CRtEurope** [comment]
The idea is great to find a key and to set keyservers, just like enigma in thunderbird. There you are able to set your own keyservers as well. I would second that, absolutely.
It would be a great advantage for administrators, too.

**Comment by alecpl** [comment]
Implemented as an automatic lookup when reading or sending mail. Future scope:
- key server discovery
- user configurable list of servers
- support for other protocols e.g. VKS, HKP v2.

**Comment by sebix** [comment]
A common and decentralized way of distributing OpenPGP keys is now the Web Key Directory: https://wiki.gnupg.org/WKD
